package.json

```diff
@@ -1,20 +1,13 @@
 {
-  "name": "loopback-next",
+  "name": "loopback-monorepo",
   "description": "Monorepo for LoopBack 4 - A highly extensible Node.js and TypeScript framework for building APIs and microservices",
-  "version": "0.1.0",
-  "private": true,
   "license": "MIT",
   "author": "IBM Corp. and LoopBack contributors",
   "copyright.owner": "IBM Corp. and LoopBack contributors",
   "repository": "github:loopbackio/loopback-next",
-  "engineStrict": true,
-  "engines": {
-    "node": "16 || 18 || 20",
-    "npm": ">=7"
-  },
   "scripts": {
     "prepare": "husky install",
-    "postinstall": "lerna bootstrap && npm run -s update-ts-project-refs",
```

Comment:
This might be the cause because lerna bootstrap is what was responsible for installing npm packages in our packages.